Ignore packages without version during prerelease (#1589)
changesets/changesets: .changeset/chilly-nights-repair.md

> ---

Reviewer: rigoro2508@gmail.com

"@changesets/assemble-release-plan": patch
"@changesets/cli": patch
---

author: @remorses
author: @vzt7

> Fixed a crash in prerelease mode when some package has no version field in its `package.json`

Reviewer 2: ```suggestion
Fixed a crash in prerelease mode when a package misses the version field in its `package.json`
```
